tests/it/main.rs

```diff
@@ -470,6 +470,15 @@ fn formatting() {
     cmd!("cargo fmt --all -- --check").run().unwrap()
 }
 
+#[test]
+fn string_escapes() {
+    setup();
+
+    assert_eq!(cmd!("\"hello\"").to_string(), "\"hello\"");
+    assert_eq!(cmd!("\"\"\" asdf\" \"").to_string(), "\"\"\" asdf\" \"");
+    assert_eq!(cmd!("\\\\\\\\\\\\\\\\\\\\\\\\\\").to_string(), "\\\\\\\\\\\\\\\\\\\\\\\\\\");
```

Comment:
Let's use the raw string literal syntax on the right hand side here `r##""""asdf""""##` to make this more readable